modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -93,15 +93,18 @@ The server ID can be found in the `serverId` column after running `SHOW SERVERS`
 |
 Defines an identical seed from an external source which will be used to seed all servers.
 
-| `seedConfig`
+| `seedConfig` label:deprecated[Deprecated in 5.26]
 | Comma-separated list of configuration values.
 |
 Defines additional configuration specified by comma-separated `name=value` pairs that might be required by certain seed providers.
-
-| `seedCredentials`
+From 5.26 it is recommended to use the `CloudSeedProvider` seed provider which does not require this configuration when seeding from cloud storage.
+For more information see xref::clustering/databases.adoc#_seed_providers[Seed providers].
+| `seedCredentials` label:deprecated[Deprecated in 5.26]
```

Comment:
```suggestion
For more information see xref::clustering/databases.adoc#_seed_providers[Seed providers].

| `seedCredentials` label:deprecated[Deprecated in 5.26]
```
since all other table rows looks like they have empty lines in between